tools/test.py

```diff
@@ -41,6 +42,12 @@ def parse_args():
         help='tmp directory used for collecting results from multiple '
         'workers, available when gpu-collect is not specified')
     parser.add_argument('--options', nargs='+', help='custom options')
+    parser.add_argument(
+        '--cfg-options',
+        nargs='+',
+        action=DictAction,
+        help='override some settings in the used config, the key-value pair '
+        'in xxx=yyy format will be merged into config file.')
```

Comment:
use an example of overwriting two options would be great

Reply:
The example is provide in [merge_from_dict](https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/config.py#L406-L418)

Reply:
Example is added